src/components/Loading.vue

```diff
@@ -21,61 +21,58 @@ import Button from "./../components/Button.vue";
 })
 export default class Loading extends Vue {
   @Prop()
-  loadingStrs!: any[] | string;
+  loaderText!: string;
+
+  @Prop()
+  loaderMultipleText!: any[];
 
   @Prop({ default: false })
   semiOpaque!: boolean;
 
   @Prop({ default: false })
   isRandom!: boolean;
 
-  strings: any = JSON.parse(JSON.stringify(this.loadingStrs));
-  loaderText: string = "";
+  displayText: string = "";
```

Comment:
I'm worried about changing the name of the prop now that 3 teams have this component integrated already - when they upgrade beaker it will break